src/utils/config/project_test.py

```diff
@@ -64,10 +82,7 @@ def test_handling_not_supported_version(version_manager: VersionManager):
         proj.load_config()
 
 
-def test_no_project_found(version_manager: VersionManager):
-    proj = Project(
-        version_manager,
-        project_root=Path(current_directory, "examples"),
-    )
+def test_no_project_found(version_manager: VersionManager, tmpdir):
+    proj = Project(version_manager, Path(tmpdir))
     with pytest.raises(NoProtostarProjectFoundError):
```

Comment:
We probably should be consistent with naming, `Error`/`Exception`, not a matter of this PR though.